.github/workflows/tests.yaml

```diff
@@ -85,3 +108,12 @@ jobs:
           fi
 
           bundle exec rake parallel:spec
+
+  tests:
+    needs:
+      - checks
+      - rspec_tests
+    runs-on: ubuntu-24.04
+    name: Test suite
+    steps:
+      - run: echo Test suite completed
```

Comment:
On second thought: this should use https://github.com/re-actors/alls-green because with skipped jobs it can give false positives. See https://github.com/theforeman/actions/blob/2a5671906614405a7480f53403d09f6db33816bb/.github/workflows/test-gem.yml#L88-L98 for an example.

Reply:
Is that really required here? All of our checks are defined in this file and they should never be skipped. And the backport job is triggered manually (and we should probably remove it, we inherited it from Perforce and it isn't working).

Reply:
> All of our checks are defined in this file and they should never be skipped. And the backport job is triggered manually (and we should probably remove it, we inherited it from Perforce and it isn't working).

Pure `needs` on a check is insufficient and this needs to always run. If a `needs` fails this may end up being skipped and then the required check isn't present. You really want to use this action and we do the same in gha-puppet: https://github.com/voxpupuli/gha-puppet/blob/b3636f92bc554e97a00b2513546db16c5035be4b/.github/workflows/basic.yml#L109-L118 & https://github.com/voxpupuli/gha-puppet/blob/b3636f92bc554e97a00b2513546db16c5035be4b/.github/workflows/beaker.yml#L207-L218

Reply:
done